modules/conflict_resolver/php/provisioners/resolvedprovisioner.class.inc

```diff
@@ -55,6 +56,9 @@ class ResolvedProvisioner extends \LORIS\Data\Provisioners\DBObjectProvisioner
               Project.ProjectID as projectid
              FROM
               conflicts_resolved
+             LEFT JOIN parameter_type
```

Comment:
I don't think this is going to work with json data (I'm not sure what the conflict_resolver or parameter_type tables look like in that case, it's possible they lie about the the tablename but I hasn't looked into it) and it's not going to take overrides into account or work if the data dictionary builder hasn't been recently run.

I think what should happen instead of trying to factor all those cases into account in the conflict_resolver, is the frontend should make one call to `/dictionary/module/instruments` (since only instruments have conflicts) and get all the field metadata, and then look up the key directly instead of making changes to the backend.